README.md

```diff
@@ -1,6 +1,35 @@
-# FRX Challenges
+# Frictionless Data Exchanges (FRX) for data challenges
 
-## The Helm chart
+Welcome to the FRX Challenges project! This is an open source repository that provides key software components for running competitive data science challenges.
+
+FRX stands for **F**rictionless **R**eproducibility e**X**change. Inspired by [Donoho, 2023](https://doi.org/10.48550/arXiv.2310.00865). This project enables communities to leverage cloud infrastructure and interactive computing tools to **host data challenges with live computation**.
```

Comment:
```suggestion
FRX stands for **F**rictionless **R**eproducibility e**X**change. Inspired by [Donoho, 2023](https://doi.org/10.48550/arXiv.2310.00865). This project enables communities to **host data challenges with live evaulation**.
```

We don't actually use any interactive computing tooling here, nor require cloud infrastructure (although we do use kubernetes, but it's not a requirement - you can put cloud back in here if you want).

We don't necessarily do live computation though - we only do live *evaluation*.